### PR TITLE
upgrade to bullet-2.83

### PR DESCRIPTION
**Pull request by AndrewMeadows**
Upgrading to Bullet-2.83, but with Huffman's CCD optimization fixes.  The question is... do all the platforms successfully build?

### CODE REVIEW

File: cmake/externals/bullet/CMakeLists.txt [hunk 1]
```diff
@@ -30,9 +30,9 @@ else ()
   ExternalProject_Add(
     ${EXTERNAL_NAME}
     #URL http://bullet.googlecode.com/files/bullet-2.82-r2704.tgz
```
Comment (Atlante45) [inline cmake/externals/bullet/CMakeLists.txt]: Should we update those 2 comments?

**Comment by hifi-gustavo** [comment]
Build console output for commit: fec187e1dd4bc8df5815ef9ae4c0560c27b4cee7 available at: https://gist.github.com/4da8d93ae5c0633c587a<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6585.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6585.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d159daf56ae77cfe2fb0e02c33d26fdea341ae67 available at: https://gist.github.com/4da8d93ae5c0633c587a<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6585.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6585.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6d7597e9fc828e7827fa349dcc08607da2f88920 available at: https://gist.github.com/4da8d93ae5c0633c587a<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6585-win64.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6585.dmg

**Comment by AndrewMeadows** [comment]
I did some basic testing and this looks good.

I verified that Huffman's continuous collision detection (CCD) optimizations were in place for "compound shapes".  The way this is done is to go into **demo** and shoot lots of ping pong balls around inside the house.  If the physics performance does not noticeably fall to its knees then it is working.

BTW, there are no new features in Bullet-2.83 that we are using yet, so there is no way to verify from the **interface** that we're using a more recent version of Bullet.  The physics system should just continue to work correctly.

**Pull request merged**